### ISSUE TITLE
Fail to validate issuer on public sts server

### ISSUE THREAD

**Issue by nikneem** [nikneem/azure-functions-jwt-binding#31]
Sometimes, the issuer is not a fixed value. For example, when allowing Microsoft Login (Any user, personal or business). In that case, you will request a token at https://login.microsoftonline.com/common/v2.0 and the issuer will be https://login.microsoftonline.com/{tenantId}/v2.0 where tenant Id is the tenant of the AD where this user lives. In case of a public login, this tenant can be any guid.

So basically this means that in this case, issuer validation will fail because only one tenant is allowed to be configured. So some kind of wildcard must be introduced so we can still validate 'some' issuer, but accept a variety of issuers.
